buildpacks/nodejs/runtime/package.json

```diff
@@ -10,7 +10,7 @@
     "test": "FUNCTION_PATH=./test/fixtures tape test/test.js | tap-spec"
   },
   "dependencies": {
-    "faas-js-runtime": "0.6.0",
+    "faas-js-runtime": "git+http://github.com/boson-project/faas-js-runtime.git#d5bfd68f4ee731f6a7170ff8978954ff8fd0c100",
```

Comment:
Is this supposed to be a version?  It looks like this might be a relic of doing some testing during development, expecting to be switched back to a version before commit.

Reply:
I see this is equivalent to the one in the `func`'s PR #355, so I assume this is not a mistake, and rather intended.  I am assuming there's some conflict/reason we can't just bump the `faas-js-runtime` minor version.

Reply:
Yes - I should have mentioned this in the PR description. The faas-js-runtime module has not been released since that change, but should be (today, probably) before this PR lands. Then this gets bumped to the new version number.